### DISCUSSION TITLE
Running Azure Linux 3.0 on Raspberry PI

### DISCUSSION THREAD

**Discussion by ohault** [microsoft/azurelinux#11203] [Q&A]
What are the current Azure Linux 3.0 experiences on Raspberry PI 4 and 5 ?

**Comment by elsaco** [comment]
There are currently several `imageconfig` files in the `toolkit` folder to build an `aarch64` image. However if you try to boot on `raspberry pi` you will be greeted by `Firmware not found` message! A device tree is need.

**Reply by AlexKeySmith** [reply]
Very interesting thanks, would it be feasible to port the firmware / device tree from Raspberry Pi OS as your everyday Linux enthusiast or is that really the remit / expertise of a distro maintainer?

**Comment by MariuszFerdyn** [comment]
Almost done for 4.0: https://www.linkedin.com/feed/update/urn:li:activity:7491098194903482369/ - stay tuned.
